app/Http/Controllers/Forum/TopicsController.php

```diff
@@ -233,11 +234,19 @@ public function show($id)
             abort($skipLayout ? 204 : 404);
         }
 
-        $postsPosition = $topic->postsPosition($posts);
+        $firstPostId = $posts->first()->post_id;
```

Comment:
is supposed to be this first post of the current thread or just of the currently loaded posts?